src/OpenLoco/src/LandscapeOptions.h

```diff
@@ -0,0 +1,135 @@
+#pragma once
+#include "EditorController.h"
+#include "Objects/Object.h"
+#include "ScenarioObjective.h"
+#include <OpenLoco/Core/EnumFlags.hpp>
+#include <cstddef>
+
+namespace OpenLoco
+{
+    enum class ScenarioFlags : uint16_t
+    {
+        none = 0U,
+        landscapeGenerationDone = (1U << 0),
+        hillsEdgeOfMap = (1U << 1),
+    };
+    OPENLOCO_ENABLE_ENUM_OPERATORS(ScenarioFlags);
+
+    enum class TopographyStyle : uint8_t
+    {
+        flatLand,
+        smallHills,
+        mountains,
+        halfMountainsHills,
+        halfMountainsFlat,
+    };
+
+    enum class LandGeneratorType : uint8_t
+    {
+        Original,
+        Simplex,
+        PngHeightMap,
+    };
+
+    enum class LandDistributionPattern : uint8_t
+    {
+        everywhere,
+        nowhere,
+        farFromWater,
+        nearWater,
+        onMountains,
+        farFromMountains,
+        inSmallRandomAreas,
+        inLargeRandomAreas,
+        aroundCliffs,
+    };
+
+#pragma pack(push, 1)
+    struct Options
```

Comment:
Wouldn't be ScenarioVariables/ScenarioParameters/Scenario be a better name? Options is way too generic in my opinion. I mean all variables are somehow tied to the Scenario, maybe just namespace it Scenario then it would be alright I guess.

Reply:
Yes agreed the name is rubbish. Just couldn't decide that's why the file is landscape options...

Reply:
I've now reworked it so that they are in the Scenario namespace